modules/search-service-api/src/main/java/org/opencastproject/search/api/SearchResultItemImpl.java

```diff
@@ -708,6 +720,7 @@ public static SearchResultItemImpl fill(SearchResultItem from) {
     item.setId(from.getId());
     item.setOrganization(from.getOrganization());
     item.setMediaPackage(from.getMediaPackage());
+    item.setAccessControlList(from.getAccessControlList());
```

Comment:
How much slower does this make things? This is used every time you invoke the search REST endpoints, isn't this? Would it make sense to make this optional? Or does it not matter?

Reply:
That's a good point but I assume it doesn't matter. For example, compared to `getMediaPackage` the line above, `getAccessControlList` is comparatively cheap. In the former, we load a big chunk of XML and parse it. Whereas in the latter, we retrieve less data from the Solr document and do fairly cheap processing on it. But maybe I just don't know about the institutions having a few MB of ACLs per event? 

Making it optional is a bit fiddly I would guess.